index.Rmd

```diff
@@ -1,14 +1,12 @@
 ---
-title: "My Final Project Template"
-author: Your name here
-subtitle: Subtitle here if desired
+title: "A Look at Single Family Housing Prices Across Buffalo" 
+author: Erik Woyciesjes
+subtitle: "2017 - 2020"
 ---
 
 # Introduction
 
-[~ 200 words]
-
-Clearly stated background and questions / hypotheses / problems being addressed. Sets up the analysis in an interesting and compelling way.
+The purpose of this project is to look at single family housing in the City of Buffalo. Using the assessed values of single family properties (City Assessment Rolls), this project aims to examine the spatial distribution and percent change of single family housing prices across the city of Buffalo. To extend this study, median household prices within neighborhoods will also be calculated to potentially illuminate trends in the changes in price of single family homes across neighborhoods. 
```

Comment:
refine to identify question, problem, or hypothesis. "look and see" is less interesting...